src/screens/General/Main.tsx

```diff
@@ -1,37 +1,31 @@
 import React from 'react'
 import EStyleSheet from 'react-native-extended-stylesheet'
 
-import { Text, View } from 'react-native'
+import { View } from 'react-native'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 
+import { DefaultText } from '../../components/Text'
```

Comment:
Please learn how to configure TS to use imports. The goal is to have import like this:

```javascript
import { nameOfImport } from 'directory'
import { nameOfImport } from 'directory/fileName'
```

Reply:
@MargaritaZagalets do you have any ideas? Cause I only know "package.json" way (like {"name":"src"} )